docs/docs/index.md

```diff
@@ -18,12 +18,11 @@ Minder can be deployed as a Helm chart and provides a CLI tool ‘minder’. Min
 * **Repo configuration and security:** Simplify configuration and management of security settings and policies across repos.
 * **Proactive security enforcement:** Continuously enforce best practice security configurations by setting granular policies to alert only or auto-remediate.
 * **Artifact attestation:** Continuously verify that packages are signed to ensure they’re tamper-proof, using the open source project Sigstore.
-* **Dependency management:** Manage dependency security posture by helping developers make better choices and enforcing controls. Minder is integrated with [Trusty by Stacklok](http://trustypkg.dev) to enable policy-driven dependency management based on the risk level of dependencies.
+* **Dependency management:** Manage dependency security posture by helping developers make better choices and enforcing controls. Minder is integrated with [OSV](https://osv.dev/) and [Trusty](https://trustypkg.dev) to enable policy-driven dependency management based on the risk level of dependencies.
 
-## Minder Cloud
+## Minder Public Instance
 
-Stacklok, the company behind Minder, provides a free-to-use SaaS version of Minder that includes a UI (for public repositories only).
-You can access Minder Cloud at [https://cloud.stacklok.com/](https://cloud.stacklok.com) and access documentation at [https://docs.stacklok.com/minder](https://docs.stacklok.com/minder).
+Stacklok, a company supporting Minder, provides a free-to-use public instance of Minder at api.stacklok.com. This instance is available for public repositories only.
```

Comment:
IMO, copy the text from this comment: https://github.com/stacklok/minder/pull/4649/files#r1790041876